library/alloc/src/fmt.rs

```diff
@@ -273,6 +273,37 @@
 //! Hello, `     123` has 3 right-aligned characters
 //! ```
 //!
+//! For the floating-point type, it employs the
+//! [roundTiesToEven](https://en.m.wikipedia.org/wiki/Rounding#Rounding_half_to_even) convention:
```

Comment:
This is unlikely to be the most appropriate link because the subsection you linked to is describing the operation of rounding a float to an integer, which is different in general from formatting a float to a particular precision. The text even calls this out: a "more sophisticated mode [is] used when rounding to significant figures".